core/src/main/java/org/apache/cxf/common/util/ClassHelper.java

```diff
@@ -28,9 +28,15 @@
  *
  */
 public class ClassHelper {
+
+    public static final String USE_DEFAULT_CLASS_HELPER = "org.apache.cxf.useDefaultClassHelper";
```

Comment:
Could you please rename it to `org.apache.cxf.useDefaultClassHelpers`  to follow similar `Spring` property name, thanks.